STM32F405RGTx/Core/Src/rtos_threads/measurements_thread.cpp

```diff
@@ -68,7 +98,11 @@ void MeasurementsThread::runMeasurements(void* args) {
     startADCandDMA();
```

Comment:
Refer to line 90 - 92, should we be calling the error handler here if the status is not HAL_OK? That's the standard for everything i've seen, unless its ok if we don't initialize this